### PR TITLE
Hawkular BTM documentation changes in support of the 0.8.0.Final release

### PR DESCRIPTION
**Pull request by objectiser**

### CODE REVIEW

**Comment by objectiser** [comment]
@theute @pilhuhn Would it be possible to do a quick review?

File: src/main/jbake/content/docs/components/btm/clientconfig.adoc [hunk 1]
```diff
@@ -1,7 +1,7 @@
 = Client Configuration Guide
 Gary Brown
 2015-06-30
```
Comment (pilhuhn) [inline src/main/jbake/content/docs/components/btm/clientconfig.adoc]: Update this?

File: src/main/jbake/content/docs/components/btm/index.adoc [hunk 2]
```diff
@@ -47,26 +59,29 @@ Instructions on how to configure the client environment can be found here: link:
...
-* Access more detailed analytics for the business transactions
+* Manage Business Transactions
+** Configure business transaction definitions
+** View a list of business transactions, with summary statistics
+** Access more detailed analytics for the business transactions
 
 These capabilities will be discussed in more detail in the following section.
 
 
 == Console
```
Comment (pilhuhn) [inline src/main/jbake/content/docs/components/btm/index.adoc:82]: I think it would be good for each of the sections to have a screen shot with annotations that are then in the text below referenced to
E.g. instead of "On the left hand side below the graph is an area for specifying filters" having that area in the screenshot marked with (1) and then writing   "1) This is an area for specifying filters..."

Comment (objectiser) [inline src/main/jbake/content/docs/components/btm/index.adoc:82]: Agree, this would be a good idea - but is it ok if it is done in a separate PR?

File: src/main/jbake/content/docs/components/btm/clientconfig.adoc (hunk 1 above)
Comment (objectiser) [inline src/main/jbake/content/docs/components/btm/clientconfig.adoc]: Have updated this and same for the other files.

File: src/main/jbake/content/docs/components/btm/index.adoc (hunk 2 above)
Comment (pilhuhn) [inline src/main/jbake/content/docs/components/btm/index.adoc:82]: On 24 May 2016, at 15:25, Gary Brown wrote:

> Agree, this would be a good idea - but is it ok if it is done in a 
> separate PR?

Sure
